src/circleci/rollcage/core.clj

```diff
@@ -221,6 +221,13 @@
                  Rollbar, 0 otherwise.
       :message - A human-readable message describing the error.
 
+  :block-fields
+  A list of fields to remove/scrub from the payload prior to sending to Rollbar
+  using snake case keywords. The list will be automatically expanded into different
```

Comment:
Can you make it clear what the input is expected to be, please? It looks like Kebab case keywords are expected.